clappr/clappr-ios
# Update fastlane

# Goal
Update the `fastlane` version we use, so that we stop seeing warnings.

This PR also includes `make help` as a task to the `MakeFile`

# How to test
1. Run `make test`
2. Check that the upgrade warnings are gone.
